model/translation/ManualTranslations.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+namespace oat\taoBackOffice\models\translation;
+
+/**
+ * Class ManualTranslations
+ *
+ * File to handle manually created translations
+ *
+ * @package oat\taoBackOffice\models\translation
+ */
+class ManualTranslations
+{
+    public function getManualTranslations()
```

Comment:
Can we please clarify why we need this file?

Reply:
yes this file needed to keep locales defined in .js files to avoid wiping them when runing locales update, @jsconan?
By default script parsing only templates/php but no js.

Reply:
I saw we only have this file for taoQtiTest extension. Are you sure we need it? We've never used this before for translations.

Reply:
As I understand from @bziondik, we only need this if you are overriding. https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/326337065/How+to+work+with+translations#How-to-override-translations